contrib/cirrus/pr-should-link-jira

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Intended for use in CI, for -rhel branches: check git PR, barf if no jira links
+#
+
+ME=$(basename $0)
+
+set -e
+
+# Github label which allows overriding this check
+OVERRIDE_LABEL="No Jira Link"
+
+# Only -rhel branches need jira links
+BRANCH_REGEX="v.*-rhel"
+
+LINK_REGEX="Fixes:? [\\t ]*https:\\/\\/issues.redhat.com\\/[\w\\/-]*"
+
+if [[ ! "${DEST_BRANCH}" =~ $BRANCH_REGEX ]]; then
+    exit 0
+fi
+
+if [[ "${CIRRUS_CHANGE_MESSAGE}" =~ $LINK_REGEX ]]; then
+    exit 0
+fi
```

Comment:
This will only match the HEAD commit.  Maybe as a followup, consider checking all commits to a PR?

Reply:
No, for a PR, CIRRUS_CHANGE_MESSAGE contains the PR description, not a commit message, and this test only runs on PRs. And, this is what we want, as then you can easily edit the description instead of having to modify a commit.

Reply:
Gotcha, sorry I assumed it was looking at commit messages.